### ISSUE TITLE
[openfl] Look into draw polys regression vs flash

### ISSUE THREAD

**Issue by azrafe7** [azrafe7/hxGeomAlgo#14]
Quite a bit since I've played with the lib, and apart from fixing the issue with latest breaking hxPixels change, now I see that the drawPoly() method doesn't work as expected (although the points are correctly computed).

flash [OK]
![image](https://user-images.githubusercontent.com/1266447/67152320-b75f2d00-f2d3-11e9-940a-e9957adedb6e.png)

html5 (and other targets) [NO]
![image](https://user-images.githubusercontent.com/1266447/67152347-263c8600-f2d4-11e9-867d-17f8e2f4ff6a.png)

**Comment by azrafe7** [comment]
UPDATE:

Mmmhh... Just commenting out the drawPaths() call for the IsoCountours example does help to show Doug-Peuck and Visv-Whyatt. May there's some issue with computed points being out of Bitmap size.

flash
![image](https://user-images.githubusercontent.com/1266447/67153026-bf729900-f2e2-11e9-9720-2a6dd9b61ecc.png)

html5
![image](https://user-images.githubusercontent.com/1266447/67153022-8f2afa80-f2e2-11e9-8ef9-5101f7b19b78.png)

**Comment by azrafe7** [comment]
Kind of a hack, but fixed in 77595d25ede256e04f02b865f851bfd282c3dd5d

Note-to-self:
```
> haxe -v
Haxe Compiler 4.0.0-rc.5+4a745347f - (C)2005-2019 Haxe Foundation

> haxelib list
actuate: [1.8.9]
asynctools: [0.1.0]
box2d: [1.2.3]
buddy: [2.10.3] git
hx3compat: [1.0.3]
hxcpp-debug-server: [dev:c:\Users\az\.vscode\extensions\vshaxe.hxcpp-debugger-1.
2.4\hxcpp-debug-server]
hxcpp: [git]
hxcs: [3.4.0]
hxjava: [3.2.0]
hxPixels: [dev:c:\az_dev\haxe\hxPixels.git\src]
layout: [1.2.1]
lime-samples: [7.0.0]
lime: [7.6.3]
openfl-samples: [8.7.0]
openfl: [8.9.5]
promhx: [1.1.0]
thx.core: [git]
utest: 1.10.1 [git]
```
